website/projects/scimaps/src/app/shared/news-item/news-item.theme.scss

```diff
@@ -0,0 +1,19 @@
+@use '~@angular/material/theming' as *;
+
+@mixin theme($theme) {
```

Comment:
This is incorrectly structured. See https://github.com/cns-iu/scimaps/blob/develop/website/projects/scimaps/src/app/core/components/footer/footer.theme.scss for correct usage